tests/LongRunning/SpeechRecoAuthTokenErrorMessageTests.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
-import got from "got";
+import bent, { BentResponse } from "bent";
```

Comment:
Does our samples use got as well and have this vulnerability?

Reply:
Thankfully no, the browser code uses XMLHTTPRequest to grab an auth token, and the example code in the JS sample README uses the axios library to send REST requests. We used the "got" lib initially bc it allows requests to set the user agent (was needed for OCSP tests). Now that OCSP tests are disabled, we can use bent, which itself has zero dependencies.